R/endpoint.R

```diff
@@ -130,11 +130,15 @@ pkgapi_endpoint <- R6::R6Class(
       tryCatch({
         data <- self$target(...)
         body <- self$returning$process(data)
+        ## Remove headers attribute as we don't want to serialize this in
+        ## the response
+        attr(body, "headers") <- NULL
```

Comment:
Couple of questions here - we could also do this in the serializer. Not sure where is more appropriate.